cypress-tests/cypress/commands/apiCommands.js

```diff
@@ -168,6 +168,8 @@ Cypress.Commands.add("apiCreateWorkspace", (workspaceName, workspaceSlug) => {
       expect(response.status).to.equal(201);
     });
   });
+  cy.logoutApi();
```

Comment:
Can we skip this part, and can we set the env from the response?

Reply:
Will address this comments in the upcoming PR's